package arenasvrd;

message Request
{
    enum Type
    {
        FIND_OPPONENT = 0;
        UPDATE_SELF = 1;
    };

    required Type type = 1;
    optional uint32 self = 2;
    optional uint32 points = 3;
};

message Response
{
    enum Status
    {
        OK = 1;
        NOT_IN_SEASON_TIME = 2;
        NO_OPPONENT = 3;
    };

    required Status status = 1;
    optional uint32 opponent = 2;
};
